netconsole/_fakeds.py

```diff
@@ -1,53 +1,51 @@
-
 import socket
 import time
 import threading
 
+
 class FakeDS:
-    '''
+    """
         Connects to the robot and convinces it that a DS is connected to it
     
         Derived from the FakeDSConnector code in GradleRIO, MIT License, Jaci R
-    '''
-    
+    """
+
     def start(self, address):
-        
+
         self.running = True
-        
-        self.tcp_socket = socket.create_connection((address, 1740), timeout=3.0)
-        self.tcp_socket.settimeout(5)
-        
+
+        # self.tcp_socket = socket.create_connection((address, 1740), timeout=3.0)
+        # self.tcp_socket.settimeout(5)
```

Comment:
Did you mean to comment this out in this PR?

Reply:
Oops.